License2Deploy/rolling_deploy.py

```diff
@@ -24,7 +24,7 @@ def __init__(self,
                session=None,
                creation_wait=[10, 60],
                ready_wait=[10, 30],
-               health_wait=[10, 30],
+               health_wait=[40, 30],
```

Comment:
Rather than changing the default for all projects can we change the default for dnbi-scoring in the jenkins job?  https://github.com/dandb/jenkins-jobs/blob/release-next/ami_workflow.dsl#L16